Files.Launcher/Program.cs

```diff
@@ -603,7 +601,7 @@ private static async Task HandleShellLibraryMessage(Dictionary<string, object> m
             {
                 case "Enumerate":
                     // Read library information and send response to UWP
-                    var enumerateResponse = await Win32API.StartSTATask(() =>
+                    var enumerateResponse = await Win32API.StartSTATask((Func<ValueSet>)(() =>
```

Comment:
Is this change necessary?

Reply:
Probably not. Oddly enough I don't even remember making this change....